build/ci/Makefile

```diff
@@ -7,6 +7,8 @@
 ROOT_DIR = $(CURDIR)/../..
 GO_MOUNT_PATH ?= /go/src/github.com/elastic/cloud-on-k8s
 
+IMAGE ?= docker.elastic.co/eck/eck-ci:$(shell md5sum $(ROOT_DIR)/operators/Gopkg.lock $(ROOT_DIR)/build/ci/Dockerfile $(ROOT_DIR)/local-volume/Gopkg.lock | awk '{print $$1}' | md5sum | awk '{print $$1}')
```

Comment:
I think we can also remove the first `awk '{print $$1}'` from this command since the command will be executed relative to the makefile path (= stable paths)?.

Test:
```diff
+hash:
+       @ md5sum $(ROOT_DIR)/operators/Gopkg.lock $(ROOT_DIR)/build/ci/Dockerfile $(ROOT_DIR)/local-volume/Gopkg.lock | md5sum
```
```bash
> m hash      
51ab277864eb84ae73e04bedacf39f23  -
                                                                                                                                                                                      
> m -C operators --no-print-directory hash 
51ab277864eb84ae73e04bedacf39f23  -
```

Reply:
Fixed